code/game/objects/items/weapons/storage/storage.dm

```diff
@@ -24,11 +24,16 @@
 	var/collection_mode = 1;  //0 = pick one at a time, 1 = pick all on tile
 	var/foldable = null	// BubbleWrap - if set, can be folded (when empty) into a sheet of cardboard
 	var/use_sound = "rustle"	//sound played when used. null for no sound.
+	var/cuff = FALSE // Used for applying cuffs briefcases. TRUE/FALSE if it has a cuff attached

```

Comment:
The fact that this is implemented at a storage level purely because that's the only level that secure and insecure briefcases share is god-awful.